server/auth/auth.go

```diff
@@ -84,6 +86,12 @@ func (client *clientInfo) suspend() {
 	client.mtx.Unlock()
 }
 
+func (client *clientInfo) resume() {
```

Comment:
Similar comment, but resume I think is not an appropriate antonym for this meaning of suspend.  I'd go with restore or reopen here.